spec/integration/alerts_spec.rb

```diff
@@ -475,11 +557,38 @@ module Hawkular::Alerts::RSpec
     end
   end
 
-  describe 'Alert/Events', :vcr do
-    VCR.configure do |c|
-      c.default_cassette_options = {
-        match_requests_on: [:method, VCR.request_matchers.uri_without_params(:startTime, :endTime)]
-      }
+  describe 'Alert/Events' do
+    before(:all) do
+      # Setup for testing
+      record('Alert/Events', credentials, 'setup') do
+        @client = Hawkular::Alerts::AlertsClient.new(ALERTS_BASE, credentials, options)
+        json = IO.read('spec/integration/alert-resources/events-test-data.json')
+        hash = JSON.parse(json)
+        hash['events'].each do |event|
+          @client.create_event event['id'], event['category'], event['text'], event['extras']
```

Comment:
Yeah. I don't remember that, I'm +1 to unify terminology with the REST API then it's easier to browse through the clients. Perhaps this "extras" parameter can be better rename.
Anyway, just a side comment, non a blocker and it can be done in a future iteration IMO.